packages/wrangler/src/index.tsx

```diff
@@ -2754,7 +2754,7 @@ export async function main(argv: string[]): Promise<void> {
       e.notes.push({
         text: "\nIf you think this is a bug, please open an issue at: https://github.com/cloudflare/wrangler2/issues/new",
       });
-      logger.error(formatMessage(e));
+      logger.log(formatMessage(e));
```

Comment:
Using `.log()` here because the message already has an `ERROR` prefix, so we don't want `.error()` to add it again.